lib/cc/engine/analyzable_files.rb

```diff
@@ -1,39 +1,29 @@
 module CC
   module Engine
     class AnalyzableFiles
-      def initialize(directory, config)
-        @directory = directory
+      def initialize(config)
         @config = config
       end
 
       def all
         @results ||= if @config["include_paths"]
-          build_files_with_inclusions(@config["include_paths"])
+          include_paths = @config["include_paths"]
+          if include_paths == ["./"]
+            ["."]
+          else
+            filter_files(@config["include_paths"])
+          end
         else
-          build_files_with_exclusions(@config["exclude_paths"] || [])
+          raise "PLACEHOLDER: We don't support exclude_paths any longer!"
```

Comment:
That's technically what it's doing since the conditional is `if @config["include_paths"]` and this is the else. This was just a quick placeholder I put in until we can decide on what to raise and say.